### ISSUE TITLE
when language-powershell is enabled "close pane" misbehaves

### ISSUE THREAD

**Issue by berdario** [jugglingnutcase/language-powershell#46]
and leave the pane empty (without any tabs inside) but still there

**Comment by jugglingnutcase** [comment]
Hi @berdario, thanks for taking the time to make a bug report here :smile: 

i don't think i understand the problem, do you have a screenshot or short gif you can send of the problem in action?

I could be wrong, but I wouldn't think that this package would be related to any pane issues since it is (for now) just a language grammar and does not really touch any of the editor's pane behaviors.

**Comment by berdario** [comment]
Yes, it's weird for me as well... I used [package-cop](https://atom.io/packages/package-cop) to pin down the culprit, and language-powershell was flagged.

After that, I tried once again to manually reproduce the problem with/without language-powershell, and I confirmed it.

Now, while trying to create a video for you, I couldn't reproduce it anymore -_-

By running the bisect again now the problem seems to be due to `ionide-fsharp`... But now I'm a bit skeptical of this result. I guess I'll use atom for a few hours more before reporting the issue to them.

Sorry for the noise
